third_party/test/proto/.bazelversion

```diff
@@ -0,0 +1 @@
+2.0.0
```

Comment:
Better do [like I did in test_scala_version](https://github.com/bazelbuild/rules_scala/commit/ecae01705dc188724950a4b6a7d2904a38a5c326#diff-af9f5a9ff6dcf95828c50792eb95701fR105) and propagate either the USER_BAZEL_VERSION in or the content of root folder `.bazelversion` and the evaluated .bazelrc flags.

Reply:
Done https://github.com/bazelbuild/rules_scala/pull/1117/commits/279a41fa8cf0cb7a74b04f09df00a1a644e7d157